src/app/containers/Toggle/toggleUtils.js

```diff
@@ -0,0 +1,7 @@
+const isToggleEnabled = (toggleName, toggles) => {
```

Comment:
Could be more concise e.g.

```jsx
const isToggleEnabled = (toggleName, toggles) => Boolean(toggles[toggleName])
```

Reply:
So the remote toggles endpoint we would eventually use, returns a object for each toggle, 

```
toggle: {
    enabled: true,
    value: 'some string'
}
```

The current method here return the entire object and the useToggle hook just pulls the `enabled` value form it. In the future we may well want to also use the `value` key.

So always returning a boolean here would not allow that.